e3db/src/main/java/com/tozny/e3db/LocalRecord.java

```diff
@@ -22,6 +23,9 @@
   private final Map<String, String> data;
   private final RecordMeta meta;
 
+  @JsonProperty("rec_sig")
```

Comment:
This annotation isn't used anywhere else - I'd rather we use them everywhere or not at all.